mlflow/server/js/.yarnrc.yml

```diff
@@ -0,0 +1,2 @@
+yarnPath: yarn/releases/yarn-3.1.1.cjs
```

Comment:
@jimmyxu-db During the sync, yarn complained that `yarn-3.1.1.cjs` doesn't exist. so I manually copied `yarn/releases/yarn-3.1.1.cjs` to suppress the error. Should `yarn-3.1.1.cjs` be included in this repo or not?

Reply:
I'm sorry! I didn't know you were going to do a copybara sync so soon. Yes, we need to copy over the yarn.cjs file manually for now because the copybara CI job only has access to files under the `mlflow/web` directory, and @apurva-koti and I wanted to unify the CI job and the actual sync script.

Reply:
Were there any other problems you ran into?